integr8ly/tutorial-web-app
# INTLY-1247: Add clipboard copy

## Motivation
[INTLY-1247](https://issues.jboss.org/browse/INTLY-1247)

## What
Replace the custom clipboard component with the PF4 version.

## Why
To be compliant with PF4.

## Verification Steps
1. Open a walkthrough and get to a page which contains copyable text.
2. Verify the following:
- You can click the button to copy the text and the text is saved to the clipboard (you can paste it somewhere)
- The copyable text in the text box is not editable
- Multiline text displays the expandable drop-down button, allowing you to display the extent of the content 

## Checklist:

## Progress

## Additional Notes
Test on your own local build or server, or feel free to use my server to test:
https://tutorial-web-app-webapp.apps.uxddev-d503.openshiftworkshop.com/

Or use my docker image on your own server:
docker.io/mfrances17/dev-tutorial-web-app:latest

**Screencap of single-line copy field:**
![single-line-copy](https://user-images.githubusercontent.com/39063664/61550630-f3f36200-aa20-11e9-8251-0df366115f8e.png)

**Screencaps of multiline copy field:**
![multi-line-copy](https://user-images.githubusercontent.com/39063664/61550659-09688c00-aa21-11e9-8446-f1a48c2d0949.png)

![expanded-multi-line-copy](https://user-images.githubusercontent.com/39063664/61550662-0cfc1300-aa21-11e9-9c23-c67e574c6ab1.png)
